### PR TITLE
lighttpd: update to 1.4.58.

### PR DESCRIPTION
**Pull request by ailiop-git**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/lighttpd/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'lighttpd'
 pkgname=lighttpd
-version=1.4.57
+version=1.4.58
```
Comment (ericonr) [inline srcpkgs/lighttpd/template:3]: .59 is out

**Comment by gstrauss** [comment]
Updating version number and checksum is trivial.
@ailiop-git would you please do so?
@ericonr can you update this PR or would you prefer that I submit a new PR?

**Comment by ericonr** [comment]
@gstrauss feel free to submit a new PR :)

**Comment by gstrauss** [comment]
I submitted #29109 for lighttpd-1.4.59

**Pull request closed without merging**